src/test/tasks.test.js

```diff
@@ -1,3 +1,112 @@
-test('', () => {
-  expect(2).toBe(2);
+import request from 'supertest';
+import app from '../app';
+import Tasks from '../models/Tasks';
+import tasks from '../seed/tasks';
+
```

Comment:
```suggestion
describe('Endpoints Test for /tasks', () => {
```
Here you can add one more describe method which will contain all of the code below